src/convenience/keyboard.ts

```diff
@@ -196,6 +196,14 @@ export class Keyboard {
      * end if necessary so you can specify more options in `reply_markup`.
      */
```

Comment:
This comment is outdated. We need to remove it and instead write something like "Returns the keyboard that was build. Note that it doesn't return resize_keyboard or other options that may be set. You don't usually need to use this method"

Reply:
Good point. We can use this PQ to do exactly that.